webui/Caddyfile-dev

```diff
@@ -35,6 +35,11 @@ proxy /api api:8180 {
   transparent
 }
 
+proxy /rpc api:8180 {
+  transparent
```

Comment:
@crashGoBoom does this need a corresponding change in Caddyfile to work in production?

Reply:
@statik yes! good call i'll work on that